pallets/protos/src/lib.rs

```diff
@@ -796,6 +797,37 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+
+		/// Delete Proto-Fragment `proto_hash` from all relevant Storage Items
+		#[pallet::weight(50_000)]
+		pub fn ban(origin: OriginFor<T>, proto_hash: Hash256) -> DispatchResult {
+			ensure_root(origin)?;
+
+			let proto_struct = <Protos<T>>::get(&proto_hash).ok_or(Error::<T>::ProtoNotFound)?;
+
+			<ProtosByCategory<T>>::mutate(&proto_struct.category, |list_protos| {
+				if let Some(list_protos) = list_protos {
+					list_protos.retain(|current_hash| proto_hash != *current_hash);
+				}
+			});
+			<ProtosByOwner<T>>::mutate(proto_struct.owner, |list_protos| {
+				if let Some(list_protos) = list_protos {
+					list_protos.retain(|current_hash| proto_hash != *current_hash);
+				}
+			});
+			for account_that_curated_proto  in <ProtoCurations<T>>::iter_key_prefix(proto_hash) {
```

Comment:
Actually I don't think we wanna remove this. It has nothing to do with get_protos anyway and this is strictly for royalties